CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+# Version 3.47.0 (2023-05-23)
+## Added
+* Support for interpolated frames to export v2
+
+## Changed
+* Removed ndjson library and replaced it with a custom ndjson parser
+## Notebooks
```

Comment:
Newline here?

Reply:
I do not think it matters for markup, but in any case I want to fix some wording so will address this too